sql/core_clients_last_seen_raw_v1.sql

```diff
@@ -0,0 +1,86 @@
+  -- Equivalent to, but more efficient than, calling udf_bitmask_range(1, 28)
+CREATE TEMP FUNCTION
+  udf_bitmask_lowest_28() AS (0x0FFFFFFF);
+  --
+CREATE TEMP FUNCTION
+  udf_shift_one_day(x INT64) AS (IFNULL((x << 1) & udf_bitmask_lowest_28(),
+      0));
+  --
+CREATE TEMP FUNCTION
+  udf_combine_days(prev INT64,
```

Comment:
since these are being used multiple places, should they be extracted into `udf/`? if so, we may need more clear names, perhaps `udf_combine_days_seen_bits` and `udf_shift_days_seen_bits` would do?

Reply:
As of #128, these will at least be codified in `udf/udf_combine_experiment_days.sql`. There's a chunk of work here to figure out how we want to handle dependent UDFs. We should also have a way to test our UDFs.

I'll create an issue to capture those needs. For now, I'm inclined to codify these only once in `udf/udf_combine_experiment_days.sql`.

Reply:
See #141